### ISSUE TITLE
Device is not online

### ISSUE THREAD

**Issue by nh13** [stfnhmplr/homebridge-synology#86]
([copied from here](https://github.com/stfnhmplr/homebridge-synology/issues/77#issuecomment-1408236026))

Error messages like:
```
[9/29/2022, 11:36:18 AM] [DiskStation] Device is not online; Polling temperature disabled.
[9/29/2022, 11:36:18 AM] [DiskStation] Device is not online; Polling disk temperature disabled.
```

The `ping` module used by this plugin routes its request to the underlying `ping` command line utility (see [here](https://github.com/stfnhmplr/homebridge-synology/blob/1c176eced7e5c2bcda494f850a93700d1ac64167/src/SynologyAccessory.ts#L108)).  Running `ping <ip address>` in the terminal gives me the error: `ping: socket: Operation not permitted`.  This lead me to looking [here](https://superuser.com/questions/1702163/ping-socket-operation-not-permitted-ubuntu-docker-image).  

The solution was to separately log into the DiskStation via ssh and execute `sudo setcap 'cap_net_raw+ep' "$(which ping)"`.  Then restart HomeBridge and voila, it can now ping the DiskStation and operate normally.  Note: I also updated the `homebridge` dependency to 1.6 just in case the API changed.

**Comment by nh13** [comment]
I think a more permanent solution is needed without the user having to manually ssh into the disk station.  Thoughts?

**Comment by stfnhmplr** [comment]
Thanks for figuring this out. The older version of this plugin did just call the diskstations webinterface to detect the online state. Reverting this back should solve this issue. What do you think?

**Comment by nh13** [comment]
@stfnhmplr that works for me!

**Comment by stfnhmplr** [comment]
I finally found some time to fix this. Instead of ping, an HTTP request is sent to the web interface to determine the online state.

**Comment by kolorfilm** [comment]
> I finally found some time to fix this. Instead of ping, an HTTP request is sent to the web interface to determine the online state.

Unfortunately this leads to another error/bug when the synology diskstation is off https://github.com/stfnhmplr/homebridge-synology/issues/89

It constantly trying the restart the whole Homebridge.

**Comment by stfnhmplr** [comment]
Hi. Thanks for reporting. Missed the error-handling part. Fixed with 1e59af8 (0.5.2)

**Comment by kolorfilm** [comment]
> Hi. Thanks for reporting. Missed the error-handling part. Fixed with [1e59af8](https://github.com/stfnhmplr/homebridge-synology/commit/1e59af85f565c91453b1e9c79d9f85252c3d86eb) (0.5.2)

Hi. You are welcome. Thanks for fixing! 🙂
